$NetBSD: patch-ac,v 1.7 2007/09/29 11:47:03 rillig Exp $

Mutual inclusion of header files is one of the funny sides of C++.

--- src/HistoryFile.h.orig	2007-09-29 13:43:51.000000000 +0200
+++ src/HistoryFile.h	2007-09-29 13:43:43.000000000 +0200
@@ -19,6 +19,8 @@
 
 #include <qstring.h>
 
+class HistoryFile; /* needed in HistoryDialog.h */
+
 #include "HistoryItem.h"
 #include "HistoryDialog.h"
 #include "tools/TextFile.h"
